public/components/integrations/components/__tests__/setup_integration.test.tsx

```diff
@@ -8,78 +8,39 @@ import Adapter from 'enzyme-adapter-react-16';
 import React from 'react';
 import { waitFor } from '@testing-library/react';
 import {
-  SetupIntegrationDataSource,
-  SetupIntegrationExistingTable,
-  SetupIntegrationMetadata,
-  SetupIntegrationNewTable,
-  SetupIntegrationStepsPage,
+  IntegrationConfig,
+  SetupIntegrationPage,
+  SetupIntegrationForm,
 } from '../setup_integration';
 
-const TEST_CONFIG = {
-  instanceName: 'Test Instance Name',
-  useExisting: true,
-  dataSourceName: 'Test Datasource Name',
-  dataSourceDescription: 'Test Datasource Description',
-  dataSourceFileType: 'json',
-  dataSourceLocation: 'ss4o_logs-test-new-location',
-  existingDataSourceName: 'ss4o_logs-test-existing-location',
+const TEST_CONFIG: IntegrationConfig = {
+  displayName: 'Test Instance Name',
+  connectionType: 'index',
+  connectionDataSource: 'ss4o_logs-nginx-test',
+};
+
+const TEST_INTEGRATION = {
+  name: 'test-integ',
+  type: 'logs',
```

Comment:
We generally keep the test constants here: https://github.com/opensearch-project/dashboards-observability/tree/main/test